beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.wdl

```diff
@@ -141,12 +145,14 @@ task emit_pe_records {
         Array[File]+ fastq2_files
         Array[Object]+ readgroups
         String fastq1_suffix = "_1.fq.gz"
-        Int mem = 2
-        Int disk_space = 10
         Int preemptible = 10
         Int max_retries = 0
         Int cpu = 1
+        Int additioinal_memory_mb = 0
+        Int additional_disk_gb = 0
     }
+    Int mem = ceil(size(fastq1_files, "MiB") + size(fastq2_files, "MiB")) + 2000 + additioinal_memory_mb
+    Int disk_space = ceil(size(fastq1_files, "GiB") + size(fastq2_files, "GiB")) + 10 + additional_disk_gb
```

Comment:
maybe use 'ceil((size([fastq1, fastq2], "GiB"))' here?  Cleaner

Reply:
That is what I had originally, but it failed in testing becasue it created an Array[Array[File]] which size cannot operate on